src/cli.js

```diff
@@ -668,7 +679,17 @@ var exports = {
       }
     });
 
-    (opts.reporter || defReporter)(results, data, { verbose: opts.verbose });
+    if (opts.hasOwnProperty('reporter')) {
+      // passed in manually 'old' way without going through entry point of script (interpret)
+      (opts.reporter)(results, data, { verbose: opts.verbose });
+    }
+    else {
+      // invoke each reporter as set up on
+      for (var index in opts.reporters) {
+        exports.getReporter(opts.reporters[index])(results, data, { verbose: opts.verbose });
```

Comment:
Can you use `for` loop instead of `for...in`? (Because `opts.reporters` is an array)